docs/pages/eas/workflows/syntax.mdx

```diff
@@ -610,15 +632,34 @@ This job outputs the following properties:
 Runs [Maestro](https://maestro.mobile.dev/) tests on a build.
 
 ```yaml
-maestro:
-  # @info #
-  type: maestro
-  # @end #
-  environment: production | preview | development # optional, defaults to preview
-  image: string # optional, defaults to 'default'. See https://docs.expo.dev/build-reference/infrastructure/ for a list of available images.
-  params:
-    build_id: string # required
-    flow_path: string | string[] # required
+jobs:
+  my_job:
+    # @info #
+    type: maestro
+    # @end #
+    environment: production | preview | development # optional, defaults to preview
+    image: string # optional, defaults to 'default'. See https://docs.expo.dev/build-reference/infrastructure/ for a list of available images.
```

Comment:
```suggestion
    image: string # optional. See https://docs.expo.dev/build-reference/infrastructure/ for a list of available images.
```
Let's not tell people what is the default. Currently it's `auto`, but I think in the future we may switch to `latest`.